db/migrate/20220908105343_adds_heading_and_chapter_id_to_goods_nomenclatures_view.rb

```diff
@@ -0,0 +1,68 @@
+Sequel.migration do
+  up do
+    run %{
+      CREATE OR REPLACE VIEW public.goods_nomenclatures
+      AS SELECT goods_nomenclatures1.goods_nomenclature_sid,
+          goods_nomenclatures1.goods_nomenclature_item_id,
+          goods_nomenclatures1.producline_suffix,
+          goods_nomenclatures1.validity_start_date,
+          goods_nomenclatures1.validity_end_date,
+          goods_nomenclatures1.statistical_indicator,
+          goods_nomenclatures1.oid,
+          goods_nomenclatures1.operation,
+          goods_nomenclatures1.operation_date,
+          goods_nomenclatures1.filename,
+          goods_nomenclatures1.path,
+          (
+              CASE WHEN "goods_nomenclatures1"."goods_nomenclature_item_id" LIKE '__00000000' THEN
+                  NULL
+              ELSE
+              LEFT ("goods_nomenclatures1"."goods_nomenclature_item_id",
+                  4)
```

Comment:
maybe on the same line?
`("goods_nomenclatures1"."goods_nomenclature_item_id", 4)`